example/tasks.py

```diff
@@ -8,8 +8,8 @@
              broker='redis://redis:6379')
 
 app.conf.update(
-    CELERY_REDIS_SCHEDULER_URL = 'redis://redis:6379',
-    CELERYBEAT_SCHEDULE={
+    redis_scheduler_url = 'redis://redis:6379',
```

Comment:
looks like when we changed to this way, the old version (celery 3/4) will not work?